javascript-practice/src/js/controllers/router.js

```diff
@@ -0,0 +1,58 @@
+import { productController } from '../main';
+
+import { ROUTES } from '../constants/url-api';
+
+import matchRoute from '../utils/matchRoute';
+
+
+/**
+ * An object that maps route names to their respective handlers
+ */
+const routes = {
+  [ROUTES.HOME]: {
+    handler: async () => {
+      await productController.displayProducts();
```

Comment:
Should `return new ProductController(model, service .....);`, not call method displayProducts here